Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/viewholders/TvSettingsViewHolder.java.orig

```diff
@@ -0,0 +1,22 @@
+package org.dolphinemu.dolphinemu.viewholders;
```

Comment:
This file should be deleted, right?

Reply:
Whoops, no idea how that happened but I've deleted the file now